pkg/cmd/chat/user/user.go

```diff
@@ -0,0 +1,192 @@
+package user
+
+import (
+	"encoding/json"
+	"errors"
+	"time"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/GetStream/stream-cli/pkg/utils"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{
+		createTokenCmd(),
+		upsertCmd(),
+		deleteCmd(),
+		queryCmd()}
+}
+
+func createTokenCmd() *cobra.Command {
```

Comment:
let's add token revocation as well

Reply:
good call but can be added in another iteration